### ISSUE TITLE
Autopilot is showing up weird and doesnt work

### ISSUE THREAD

**Issue by Mushr00m2010** [Qantas94Heavy/autopilot-pp#34]
my autopilot shows up very weird ive installed it again but its still the same can someone fix this 
![PROBLEEM 1](https://github.com/user-attachments/assets/869edc03-91a4-4bee-982e-68f75f0eb989)
![PROBLEEM2](https://github.com/user-attachments/assets/baa908ad-affe-477b-85da-c0b01614dcaf)
please i just got a yoke and i want to fly yesterday it was fine but today its not anymore please help me

**Comment by GripGeoFs** [comment]
I have exactly the same problem.

**Comment by Mushr00m2010** [comment]
it probably has something to do with a update in geofs but idk how to fix or code

**Comment by CharlieDelta28** [comment]
Same problem here, and it's definitely game breaking. It seems that has to do with the fact that it can't chose for a specific mode (LAT/LON, WAYPOINT or HEADING) and thus displays them all. The "Autopilot engage" button (and the change mode button) dont work either. Is this just the case of updating the display ?. Anyway just a few ideas from what i have seen so far but I am no expert.

@Qantas94Heavy I know you are probably quite busy... could you have a look ? Many thanks on behalf of the community.

**Comment by PatrickJYKang** [comment]
Same issue here

**Comment by Ilirian2007** [comment]
Same problem with that weird format glitch

**Comment by JayBhaskarCoding** [comment]
Yep I am also facing the same problem. I tried to update the code but to no avail. @Qantas94Heavy has done an amazing and mind blowing job at making this plugin and that sure can be seen from the code. So it sure is very hard to find something that might have went wrong. We know you are doing your best to resolve this.
